src/Microsoft.Azure.WebJobs.Host/Executors/DefaultStorageCredentialsValidator.cs

```diff
@@ -88,5 +96,23 @@ internal class DefaultStorageCredentialsValidator : IStorageCredentialsValidator
                 throw;
             }
         }
+
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Naming", "CA2204:Literals should be spelled correctly", MessageId = "IsDevStoreAccount")]
+        internal static bool GetIsDevStoreAccountFromCloudStorageAccount(CloudStorageAccount account)
+        {
```

Comment:
Instead of private reflection, can we check for the emulator account URL somehow, instead?

Reply:
I had thought of this, but hesitated since we don't have control of that URL and it could change in the future. If it's preferable to private reflection, though, I'll go ahead with it.

Reply:
As I'm checking the [emulator overview docs](https://docs.microsoft.com/en-us/azure/storage/common/storage-use-emulator), it also looks like there's a way to use the `devstoreaccount1` account name, and even a way to specify a development proxy. So I guess it's not as simple as just checking a URL as I thought it would be.

I think it's okay to do private reflection as long as our test hits this code path (via a 'real' e2e manner like I commented elsewhere). That will alert us pretty quickly if the private reflection has changed.